### ISSUE TITLE
Project website has no IPv6 DNS record

### ISSUE THREAD

**Issue by tambry** [viewvc/viewvc#149]
[The website](http://viewvc.org) has no AAAA DNS record, meaning that the website is inaccessible on IPv6-only networks and has worse accessibility on IPv6-optimized networks.

An AAAA record should be added to the DNS by whoever admins the site.

**Comment by cmpilato** [comment]
This is _technically_ off-topic, but I'll keep it around as a reminder.

**Comment by tambry** [comment]
Seems like the fix would actually be as simple as adding `2600:3c00::f03c:91ff:fe7e:d702` as an AAAA record.

**Comment by cmpilato** [comment]
@tambry How did you determine that IPv6 address? We recently -- like, _yesterday_ recently -- moved viewvc.org to a new server, but all I knew was the new IPv4 address for the box.

**Comment by tambry** [comment]
@cmpilato By default Linode's IPv4 addresses have a RDNS record for the server-specific subdomain, which itself has the default IPv4 and IPv6 address of that instance.

So like this:
```
[root@tambre ~]# host viewvc.org
viewvc.org has address 45.79.25.59
viewvc.org mail is handled by 50 sanpietro.red-bean.com.
[root@tambre ~]# host 45.79.25.59
59.25.79.45.in-addr.arpa domain name pointer li1124-59.members.linode.com.
[root@tambre ~]# host li1124-59.members.linode.com
li1124-59.members.linode.com has address 45.79.25.59
li1124-59.members.linode.com has IPv6 address 2600:3c00::f03c:91ff:fe7e:d702
```

Fortunately your nginx already seems to be listening on IPv6, so only the AAAA record is missing.

**Comment by cmpilato** [comment]
I'll add the AAAA record now!

**Comment by tambry** [comment]
And your Linode's network configuration should be nicely listed under the networking tab:

![screenshot of a Linode's networking tab with the IPv6 section highlighted](https://user-images.githubusercontent.com/1827708/60282346-689a1b80-990f-11e9-932a-f1e175c2e345.png)

**Comment by tambry** [comment]
Working now 👍

**Comment by cmpilato** [comment]
Yeah, I just stumbled across that information, too.  Either way, the AAAA's have been added for viewvc.org and viewvc.com, now.  Thanks!
